src/sbt-test/sbt-sass-compiler/can-compile-if-dart-sass-executable-missing/build.sbt

```diff
@@ -0,0 +1,32 @@
+import java.nio.file.{Files, Paths}
+import scala.sys.process._
+
+lazy val tempDir = Paths.get("/tmp/de.larsgrefer.sass.embedded.connection.BundledPackageProvider")
```

Comment:
this will need to be the path that it unpacks the dart-sass executable which I think will be defined in BundledPackageProvider - but might not be readable from it so might have to hardcode it